config/initializers/mock_ldap_config_check.rb

```diff
@@ -0,0 +1,19 @@
+should_use_mock_ldap = ::EnvironmentVariable.is_true('USE_TEST_PALS_LDAP')
```

Comment:
What do you think about moving this to a class method on `MockLdap` and calling it here?  Then `MockLdap` could call that method each time as well, and you could remove the env check there.  That would also move all the things read the env related to this to the same file.